multiapps-controller-client/src/main/java/org/cloudfoundry/multiapps/controller/client/lib/domain/BindingDetails.java

```diff
@@ -0,0 +1,41 @@
+package org.cloudfoundry.multiapps.controller.client.lib.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BindingDetails {
+    private String bindingName;
+    private Map<String, Object> config;
+
+    public BindingDetails() {
+
+    }
+
+    public BindingDetails(String bindingName, Map<String, Object> config) {
+        this.bindingName = bindingName;
+        this.config = config;
+    }
+
+    public String getBindingName() {
+        return bindingName;
+    }
+
+    public Map<String, Object> getBindingParameters() {
+        return config;
+    }
+
+    public void setBindingName(String bindingName) {
+        this.bindingName = bindingName;
+    }
+
+    public void setBindingParameters(Map<String, Object> config) {
+        this.config = config;
+    }
+
+    public void addBindingParameter(String name, Object value) {
+        if (this.config == null) {
+            this.config = new HashMap<>();
+        }
+        this.config.put(name, value);
```

Comment:
you do not need this here

Reply:
if the method is only used for test it is better to remove it and pass the already created config parameters.
I suggest to make the class immutable